File: Questions/questions/arrays/MinAndMax.java
public class MinAndMax{
    static int min(int[] arr){
        int min = arr[0];
        for (int j : arr) {
            if (min > j) {
                min = j;
            }
        }
        return min;
    }

    static int max(int[] arr){
        int max = arr[0];
        for (int j : arr) {
            if (max < j) {
                max = j;
            }
        }
        return max;
    }

    public static void main(String[] args) {
        int[] arr = {4,2,4,90,6, 83};
        System.out.println(min(arr));
        System.out.println(max(arr));
    }
}

File: golang/arrays/minandmax.go
package main

import "fmt"

func main() {
	arr := []int{1, 2, 53, 4, 5, 6}
	fmt.Println(arr)
	min, max := findMinMax(arr)
	fmt.Println("min: ", min, "max: ", max)
}

func findMinMax(arr []int) (int, int) {
	min := arr[0]
	max := arr[0]
	for i := 1; i < len(arr); i++ {
		if arr[i] < min {
			min = arr[i]
		}
		if arr[i] > max {
			max = arr[i]
		}
	}
	return min, max
}
